aws/cloudformation/bootstrap_frontend.sh.erb

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+# UserData bootstrap script for CloudFormation stack instances.
+# Note: Every time this script's contents change,
+# the LaunchConfiguration will be replaced in the CloudFormation stack.
+
+STACK=<%=stack_name%>
+REGION=<%=region%>
+INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
```

Comment:
yes, it's the hard-coded IP for [EC2 instance metadata](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html).